packages/embark-ui/src/components/Blocks.js

```diff
@@ -14,7 +14,8 @@ const Blocks = ({blocks, changePage, currentPage, numberOfPages}) => (
           <h2>Blocks</h2>
         </CardHeader>
         <CardBody>
-          {blocks.map(block => (
+          {!blocks.length && "No blocks to display"}
+          {blocks.length && blocks.map(block => (
```

Comment:
```suggestion
          {blocks.map(block => (
```
The map does the equivalent of  a length check

Reply:
Good point, will change it as suggested.

Reply:
Done.